opentracing-okhttp3/pom.xml

```diff
@@ -17,6 +17,7 @@
     <dependency>
       <groupId>io.opentracing</groupId>
       <artifactId>opentracing-noop</artifactId>
+      <scope>provided</scope>
```

Comment:
Could you please rather remove noop? I am not sure why it is pulled in, I don't think it's used anywhere.

Reply:
Removed!